docs/source/concepts/functionality/command-line/command_line_autodoc.rst

```diff
@@ -0,0 +1,12 @@
+:orphan:
+
+.. dropdown:: Distribution Statement
+
+ This source code is protected under the license referenced at
+ https://github.com/NRLMMD-GEOIPS.
```

Comment:
I think the latest verbiage for this is "subject to" not "protected under"

Reply:
Fixed